thoth/common/openshift.py

```diff
@@ -526,6 +540,43 @@ def _transform_cronjob_to_job(item: Dict[str, Any]) -> Dict[str, Any]:
         job_spec["metadata"]["generateName"] = item["metadata"]["name"] + "-"
         return job_spec
 
+    def _submit_workflow(
+        self,
+        wf: argo.models.V1alpha1Workflow,
+        namespace: Optional[str] = None,
+        parameters: Optional[Dict[str, str]] = None,
+    ):
```

Comment:
```
) -> None:
```

or, in a better case, a workflow id?

Reply:
Correct, I changed that to `workflow.name` in the meantime and we can discuss potentially more sophisticated approach (i.e. hash or generating id similar to the inspections) if needed.